@xen-orchestra/lite/src/views/vm/VmHeader.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <TitleBar :icon="faDisplay">
+    {{ name }}
+    <template #actions>
+      <AppMenu horizontal>
+        <MenuItem :icon="faPowerOff" color="info">
+          {{ $t("change-state") }}
+          <template #submenu>
+            <MenuItem :icon="faPlay" :disabled="!isHalted">{{
+              $t("start")
+            }}</MenuItem>
+            <MenuItem :icon="faServer" :disabled="!isHalted"
+              >{{ $t("start-on-host") }}
+              <template #submenu>
+                <MenuItem
+                  v-for="host in hostStore.allRecords"
+                  v-bind:key="host.$ref"
+                  :icon="faServer"
+                >
+                  <div class="wrapper">
+                    {{ host.name_label }}
+                    <div>
+                      <UiIcon
+                        class="star"
+                        :icon="
+                          host.$ref === poolStore.pool?.master
+                            ? faStar
+                            : undefined
+                        "
```

Comment:
Do you know why there is no style after this line? It's a little hard to read your PR after that line.

Please, can you fix it?

Reply:
I think it's a problem of github with the syntax of `vuejs`